pyproject.toml

```diff
@@ -2,7 +2,7 @@
 requires = [
   "setuptools == 67.7.2",
   "setuptools_scm[toml] == 7.1.0",
-  "frequenz-repo-config[api] == 0.2.0",
+  "frequenz-repo-config[api] == 0.3.0",
```

Comment:
Is this change required before making new releases?

Reply:
Okay, now that the CI run has finished, I can see that some of the tests are failing. So this probably needs to be removed.

But this is interesting. When we tag v0.3.0, this will still point to v0.2.0. Will this cause a problem?

Reply:
Removed.